boxoffice/views/__init__.py

```diff
@@ -1,4 +1,2 @@
-import boxoffice.views.setup
-import boxoffice.views.index
-import boxoffice.views.login
-import boxoffice.views.registration
+from boxoffice.views.item_collection import *
+from boxoffice.views.order import *
```

Comment:
You don't need to `import *` for views. This is sufficient: `from . import item_collection, order, …`